check50.py

```diff
@@ -32,8 +33,8 @@ def main():
     parser.add_argument("files", nargs="*")
     parser.add_argument("-d", "--debug", action="store_true")
     parser.add_argument("-l", "--local", action="store_true")
-    parser.add_argument("-f", "--force", action="store_true")
     parser.add_argument("--log", action="store_true")
+    parser.add_argument("--no-autoupdate", action="store_true")
```

Comment:
I wonder if we should flip this to be `--update` and then alias `check50` to `check50 --update` on CS50 IDE, omitting that alias from CS50 IDE Offline? In one or the other, we'll need to disable. Is either better?